capstone/src/main/angular/data_demo/src/styles.css

```diff
@@ -65,4 +65,26 @@
 
 .binary_file span {
     flex-grow: 1;
+}
+
+input[type=search] {
+
+    width: 100px;
+    height: 43px;
+    border: 2px solid #ccc;
+    border-radius: 15px;
+    font-size: 16px;
+    background-color: white;
+    -webkit-transition: width 0.4s ease-in-out;
+    transition: width 0.4s ease-in-out;
+
+}
+
+input[type=search]:focus {
+    width: 110%;
+    height: 43px;
+}
+
+input::-webkit-calendar-picker-indicator {
```

Comment:
What's going on here?

Reply:
According to https://trac.webkit.org/wiki/Styling%20Form%20Controls it disables the "native date picker", but why do we want to disable that? This would only affect webkit browsers (so not Safari or Firefox) and may alter user's expectations of date pickers. Do we have a date picker on the site?

Reply:
Nope, because the search button has a datalist created for it, it show a drop down button which I didn't want to display. I will comment on that.